src/main/java/twilightforest/integration/TFBaublesIntegration.java

```diff
@@ -42,4 +82,21 @@ public static ItemStack[] keepAllBaubles(EntityPlayer player) {
         return baublesInventory;
     }
 
+    public static void restoreBaubles(EntityPlayer player, ItemStack[] baublesInventory) {
+        if (TwilightForestMod.areBaublesLoaded && baublesInventory != null) {
+            InventoryBaubles inventoryBaubles = PlayerHandler.getPlayerBaubles(player);
+            for (int i = 0; i < inventoryBaubles.getSizeInventory(); i++) {
+                if (baublesInventory[i] != null) inventoryBaubles.setInventorySlotContents(i, baublesInventory[i]);
+            }
+        }
+    }
+
+    public static void playerSaveDo(EntityPlayer player, File directory) {
+        PlayerHandler.savePlayerBaubles(
```

Comment:
You probably want a `TwilightForestMod.areBaublesLoaded` check here like you have for the other methods.